DS13/code/modules/mob/living/carbon/human/species_types/t800.dm

```diff
@@ -0,0 +1,225 @@
+	var/list/options = list("assasinate", "protect", "clear")
+	for(var/option in options)
+		options[option] = image(icon = 'DS13/icons/actions/weaponselect.dmi', icon_state = "[option]")
+	var/dowhat = show_radial_menu(user,src,options)
+	if(!dowhat)
+		return
+	var/list/targets = list()
+	for(var/mob/M in GLOB.player_list)
+		if(M && is_station_level(M.z) && M.mind)
+			targets += M
+	switch(dowhat)
+		if("assasinate")
+			var/mob/selection = input("Select a new target", "Neural net processor", null, null) as null|anything in targets
+			to_chat(user, "<span class='warning'>Extermination target set. Re-insert neural net processor to complete reprogramming...</span>")
+			if(!selection)
+				return
+			assasinate_target = selection
+		if("protect")
+			var/mob/selection = input("Who do you want this unit to protect?", "Neural net processor", null, null) as null|anything in targets
+			to_chat(user, "<span class='warning'>Target set. New directive: protect [selection]. Re-insert neural net processor to complete reprogramming...</span>")
+			if(!selection)
+				return
+			protect_target = selection
+		if("clear")
+			to_chat(user, "<span class='warning'>All targets reset.</span>")
+			protect_target = null
+			assasinate_target = null
+
+/obj/item/organ/body_egg/t800chip/Remove(mob/living/carbon/M, special = 0) //Allows for deborging
+	. = ..()
+	to_chat(M, "<span class='warning'>WARNING: Neural net processor removed. Targetting parameters reset. Directive: Await new target.</span>")
+	if(objective_holder)
+		objective_holder.objectives = list()
+		objective_holder = null
+	if(M.mind)
+		M.mind.remove_antag_datum(/datum/antagonist/custom)
+
+/obj/item/organ/body_egg/t800chip/Insert(mob/living/carbon/M, special = 0) //Allows for deborging
+	. = ..()
+	if(!M.mind)
+		return
+	if(!objective_holder)
+		objective_holder = M.mind.add_antag_datum(/datum/antagonist/custom)//To hold objectives.
+	if(protect_target)
+		var/datum/objective/protect/X = new
+		X.owner = M.mind
+		if(protect_target.mind)
+			X.target = protect_target.mind
+		X.explanation_text = "Protect [protect_target] at all costs. Accept any commands they give you..."
+		objective_holder.objectives += X
+		to_chat(M, "<span class='warning'>Mission parameters modified: Protect [protect_target]</span>")
+	if(assasinate_target)
+		var/datum/objective/assassinate/X = new
+		X.owner = M.mind
+		if(assasinate_target.mind)
+			X.target = assasinate_target.mind
+		X.explanation_text = "Terminate [protect_target] at all costs. Casualties are acceptable."
+		objective_holder.objectives += X
+		to_chat(M, "<span class='warning'>Mission parameters modified: New target: [assasinate_target]</span>")
+	to_chat(M, "<span class='warning'>WARNING: Neural net processor connection re-established.</span>")
+
+/mob/living/carbon/human/species/t800
+	race = /datum/species/t800
+
+/mob/living/carbon/human/species/t800/adjust_hygiene(amount)
+	return //Fuck off stinky robot
```

Comment:
Have it at a very reduce amount going in instead. So its much harder to get dirty but can after a very very long period of time.

Reply:
Robots dont develop stank